### PR TITLE
ZOO-1935 Review the batch guide and reference

### PR DESCRIPTION
**Pull request by ansobdx**
Review the guide and reference for batch processing. Update the text and examples based on the feedback from Ben Cameron

### CODE REVIEW

File: source/docs/guides/batch_payment/index.md [hunk 1]
```diff
@@ -15,56 +15,58 @@ navigation:
 
 # Batch payment
 
-The Batch Payment API allows you to batch transactions together into a single a API call.
+The Batch Payment API allows you to combine multiple transactions into a single API request.
 
-You can batch credit card transactions and bank to bank (ACH/EFT) transactions. You can use bank to bank transactions to push funds to a creditor, or pull funds from a debtor. The most common use cases for bank to bank transactions is payroll or Accounts Payable, and synchronized subscription models where multiple people are charged on a certain day each month, or each year.
+The Batch Payment API supports credit card and bank-to-bank (ACH/EFT) transactions. Each payment type must be processed through their own batch request. Within a single bank-to-bank request, you can pay (credit) and collect (debit) funds.
```
Comment (mikehuffman) [inline source/docs/guides/batch_payment/index.md]: Each payment type must be processed with a separate batch request.

File: source/docs/guides/batch_payment/index.md [hunk 2]
```diff
@@ -15,56 +15,58 @@ navigation:
...
+The Batch Payment API allows you to combine multiple transactions into a single API request.
 
-You can batch credit card transactions and bank to bank (ACH/EFT) transactions. You can use bank to bank transactions to push funds to a creditor, or pull funds from a debtor. The most common use cases for bank to bank transactions is payroll or Accounts Payable, and synchronized subscription models where multiple people are charged on a certain day each month, or each year.
+The Batch Payment API supports credit card and bank-to-bank (ACH/EFT) transactions. Each payment type must be processed through their own batch request. Within a single bank-to-bank request, you can pay (credit) and collect (debit) funds. 
 
-The Batch Payment API is compatible with our Payment Profiles service. This allows you to securely store credit card and bank account details on our servers.
+## Use case
 
-The Batch Report API allows you to query the status of batches and individual transactions.
+1. A gym software can collect membership fees from customers of multiple gyms and submit as one API call. Payment can be made from a credit card or directly from their bank account.
```
Comment (mikehuffman) [inline source/docs/guides/batch_payment/index.md]: gym management software

Customers can make payments from their credit card or directly from their bank account.

File: source/docs/guides/batch_payment/index.md [hunk 3]
```diff
@@ -15,56 +15,58 @@ navigation:
...
+1. An accounting department can submit payroll and expense repayment to its employees.
```
Comment (mikehuffman) [inline source/docs/guides/batch_payment/index.md]: expense reimbursement

File: source/docs/guides/batch_payment/index.md [hunk 4]
```diff
@@ -15,56 +15,58 @@ navigation:
...
+### Processing
 
-Batches of bank to bank transactions take multiple days to process. Batches submitted before 1100 PST (AM) can be scheduled to begin processing on the same day. Batches submitted after this time will be processed the following day.
+Credit card transactions can be processed in real-time, or you can schedule them to process on a specific date. If you select a future date, those transactions will be processed at 6:00 AM (PST), on the date you’ve selected.
 
-### Funds transfers
+### Settlement
 
-We process funds transfers in 2 steps. We pull funds from the payer's bank account into our bank account and then we push them to the payee's bank account.
+Settlement for credit card batches is dependent on the card type and processor. Typically people see a three-day settlement lag. Higher risk accounts may have a longer settlement time.
```
Comment (mikehuffman) [inline source/docs/guides/batch_payment/index.md]: Typically a three day settlement lag can be expected.

File: source/docs/guides/batch_payment/index.md [hunk 5]
```diff
@@ -15,56 +15,58 @@ navigation:
...
+Credit card and bank-to-bank transactions have different processing flows, settlement times, and fees.
 
-## Settlement flow
+## Credit cards
 
-Batches of credit card transactions can be processed immediately, on upload, or they can be scheduled to be processed on a certain date. Scheduled batches of credit card transactions are processed at 0600 PST (AM) each day.
+### Processing
 
-Batches of bank to bank transactions take multiple days to process. Batches submitted before 1100 PST (AM) can be scheduled to begin processing on the same day. Batches submitted after this time will be processed the following day.
+Credit card transactions can be processed in real-time, or you can schedule them to process on a specific date. If you select a future date, those transactions will be processed at 6:00 AM (PST), on the date you’ve selected.
```
Comment (mikehuffman) [inline source/docs/guides/batch_payment/index.md]: will be processed at 3:00 AM (PST)

File: source/docs/guides/batch_payment/index.md [hunk 6]
```diff
@@ -15,56 +15,58 @@ navigation:
...
 
-We process funds transfers in 2 steps. We pull funds from the payer's bank account into our bank account and then we push them to the payee's bank account.
+Settlement for credit card batches is dependent on the card type and processor. Typically people see a three-day settlement lag. Higher risk accounts may have a longer settlement time. 
 
-- Eg. You are a fitness club charging your members for the monthly fees: members are the payers, money is pulled from their accounts, and you are the payee, the recipient of the money.
-- Eg: You want to pay your vendors: you are the payer, and the vendors are the payees.
+### Fees
 
-We add a lag between receiving and re-sending the funds to mitigate the risk associated with returned transfers. This lag is usually 3 to 5 business days and is set relative to the risk associated with your business. Funds transfers can be returned due to closed or invalid bank accounts, insufficient funds, or disputes. We usually receive returns within 2 to 4 business days.
+Credit card fees will depend on your account but will include a percentage and a flat fee. It's handled through the regular credit card settlement process.
```
Comment (mikehuffman) [inline source/docs/guides/batch_payment/index.md]: Credit card fees will depend on your account but a percentage and a flat fee will apply. 

Settlement for batch credit card transactions is handled in the same way as the regular credit card settlement process.

File: source/docs/guides/batch_payment/batch.md [hunk 7]
```diff
@@ -19,59 +19,59 @@ A batch request is a single API call to upload a file containing a list of trans
...
 
 `Authorization: Passcode Base64Encoded(merchant_id:passcode)`
 
 ### Authorizing as an Integrated Software Vendor (ISV)
 
 If you have a partner account with us, you can specify the sub-merchant account on which to process the batched transactions.
 
-- Authorize the API request as by passing your merchant ID and API passcode in the Authorization header.
-- Specify the sub-merchant account id in the body.
+- Authorize the API request as by passing your ISV parent merchant ID and API passcode in the Authorization header.
```
Comment (mikehuffman) [inline source/docs/guides/batch_payment/batch.md]: Authorize the API request by passing your ..

File: source/docs/guides/batch_payment/batch.md [hunk 8]
```diff
@@ -19,59 +19,59 @@ A batch request is a single API call to upload a file containing a list of trans
...
+The server's response is a single HTTPS response using the text content type. The response body is formatted as JSON and refers to the success of the upload of the batch request itself, not the success of the individual transactions it contained.
 
 You can query individual transactions from a batch through the Batch Payment Report API and the Report API.
 
-The response object contains a "code" property indicating the success of the request. This will be a number between 1 and 23, inclusive, where "1" indicates success. It has a "message" property with a description of the code. It also has a "process_date" property indicating the date that the batch will be sent to the bank to begin processing.
+The response object contains a "code" property indicating the success of the request. This will be a number between 1 and 23, inclusive, where "1" indicates success. It has a "message" property with a description of the code. It also has a "process_date" property indicating the date that the batch will be sent to the bank to begin processing. The "batch_mode" property indicates if your account is in test or is live. The data is sent to the bank only for accounts live.
```
Comment (mikehuffman) [inline source/docs/guides/batch_payment/batch.md]: The data is sent to the bank only for accounts that are live.

File: source/docs/guides/batch_payment/batch.md [hunk 9]
```diff
@@ -19,59 +19,59 @@ A batch request is a single API call to upload a file containing a list of trans
...
 
 ## Examples
 
-This section includes a sample HTTP request that references a file to be uploaded and a sample request where the data is declared inline within the request. It also includes formatted sample data for batched funds transfer and credit card payments.
+This section includes a sample HTTPS request that references a file to be uploaded and a sample request where the data is declared inline within the request. It also includes formatted sample data for batched funds transfer and credit card payments.
```
Comment (mikehuffman) [inline source/docs/guides/batch_payment/batch.md]: It also includes formatted sample date for bank-to-bank transfer and credit card payments.

File: source/docs/references/batch_payment_report/index.md [hunk 10]
```diff
@@ -81,7 +81,7 @@ Filters are assembled as in groups containing a column name (`rptFilterBy*`), an
 
 The format of the response object is relative to `rptFormat` element in the request in either JSON, XMl, TSV, CSV, or XLS formats.
 
-JSON and XMl response object contains a "code" property" indicating the success of the request. This will be a number between 1 and 8, inclusive, where "1" indicates success. It also has a message property with a description of the code.
+JSON and XML response object contains a "code" property" indicating the success of the request. This will be a number between 1 and 8, inclusive, where "1" indicates success. It also has a message property with a description of the code.
```
Comment (mikehuffman) [inline source/docs/references/batch_payment_report/index.md]: Not sure , but the contains a "code" property" looks like there is an extra quote in the middle

File: source/docs/guides/batch_payment/batch.md (hunk 7 above)
Comment (ansobdx) [reply source/docs/guides/batch_payment/batch.md]: made changes

**Pull request merged**